pymatgen/io/vasp/sets.py

```diff
@@ -1215,6 +1215,73 @@ def from_prev_calc(cls, prev_calc_dir, **kwargs):
         return input_set.override_from_prev_calc(prev_calc_dir=prev_calc_dir)
 
 
+class MatPESStaticSet(MPStaticSet):
+    """Creates input files for a MatPES static calculation."""
+
+    CONFIG = _load_yaml_config("MatPESStaticSet")
+
+    def __init__(
+        self,
+        structure,
+        functional=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            structure (Structure): Structure from previous run.
+            functional (str): We can set the VASP functional to R2SCAN or R2SCAN+U or PBE+U, or it will
+                generate input sets PBE functional.
+            **kwargs: kwargs supported by MPRelaxSet.
+        """
+        super().__init__(structure, MatPESStaticSet.CONFIG, **kwargs)
+        kwargs.setdefault("user_potcar_functional", "PBE_54")
+        if isinstance(functional, str):
```

Comment:
What is this isinstance for? I don't think functional should ever be None. Just set the functional default to PBE.